.github/workflows/install-test-react-native.yml

```diff
@@ -18,6 +18,7 @@ defaults:
 
 jobs:
   install:
+    name: Install Test on React Native ${{ matrix.platform == 'ios' && 'iOS' || 'Android' }} realm@${{ matrix.realm-version }}, react-native@${{ matrix.react-native-version }} new architecture ${{ matrix.new-architecture && 'enabled' || 'disabled' }} running ${{ matrix.engine }}
```

Comment:
Making the name of the job a bit more readable:

![Uploading Screenshot 2023-02-06 at 13.17.36.png…]()